docs/02-lazy-services.md

```diff
@@ -0,0 +1,2 @@
+## Lazy services
+To allow for publishers to be lazy loaded you can use [Symfony's proxy manager bridge](http://symfony.com/doc/current/components/dependency_injection/lazy_services.html).
```

Comment:
Since the extension explicitly makes the service lazy (it's not optional), the dependency should be added to `composer.json`. Maybe it's a good idea to put this in a compiler pass, so it does become optional?